doc/man/8/ceph.rst

```diff
@@ -1564,6 +1566,32 @@ Usage::
 
 	ceph pg stat
 
+A second format for PG-directed commands is the following:
+
+Using the ``tell`` command with a PG identifier. This sends a command to the OSD
+that is currently acting as the primary for the specified PG.
+
+Subcommand ``scrub`` starts a scrub of <pgid>, similar
+to ``ceph pg scrub <pgid>``.
+
+Usage::
+
+	ceph tell <pgid> scrub
+
+Subcommand ``deep-scrub`` starts a deep-scrub of <pgid>, similar
+to ``ceph pg deep-scrub <pgid>``.
+
+Usage::
+
+	ceph tell <pgid> deep-scrub
+
+Subcommand ``scrub-abort`` aborts any ongoing scrub of <pgid>, and removes
+any previous scrub or deep scrub opetrator request for the PG.
+
+Usage::
+
+        ceph tell <pgid> scrub-abort
```

Comment:
I suggest to indent this line similarly to the others, for example [line 1586](https://github.com/ceph/ceph/pull/66626/files#diff-51b442383774215f88eed99c44783afd456d62a7d28583c65620a6984acd84f1R1586)